### DISCUSSION TITLE
Error While sending Email

### DISCUSSION THREAD

**Discussion by arpit456jain** [vigneshshettyin/Flask-Generate-Certificate#440] [General]
<img width="960" alt="error2" src="https://user-images.githubusercontent.com/55352601/119487497-140baf00-bd77-11eb-8804-cf532c68a0ff.png">

@vigneshshettyin i am getting this error when trying to register with a new account can you help me out

**Comment by vigneshshettyin** [comment]
Activate your account by changing the status flag to 1 using DB Browser. (https://sqlitebrowser.org/dl/) @arpit456jain

**Reply by arpit456jain** [reply]
Yeah @vigneshshettyin i know i can do that but i want to know why the email is not sending.
Actually i am working on a new feature 

Email warning to me if someone else is trying to login in my account and he/she fails more than 5 times.

So for checking this feature that email should be working properly

**Reply by vigneshshettyin** [reply]
> Yeah @vigneshshettyin i know i can do that but i want to know why the email is not sending.
> Actually i am working on a new feature
> 
> Email warning to me if someone else is trying to login in my account and he/she fails more than 5 times.
> 
> So for checking this feature that email should be working properly

No, we don't need this feature. @arpit456jain
